tests/util_tests/test_shotgun_path.py

```diff
@@ -255,3 +255,17 @@ def test_current_platform_file(self):
             "Cannot resolve file name - unsupported os platform 'potato'"
         ):
             ShotgunPath.get_file_name_from_template("/%s.yml", "potato")
+
+    def test_hashing(self):
+        """
+        Ensures two ShotgunPath with the same path generate the same hash.
```

Comment:
please add a note that we are doing this test because of py2/py3!

Reply:
Well, it's not really because of compatibility issues that I am doing this. As you've noticed elsewhere I've not always chosen to implement the __hash__ method because it didn't feel relevant. But this is part of the public API and it makes sense to allow hashing of file paths.